src/qiboml/backends/jax.py

```diff
@@ -1,25 +1,50 @@
 from qibo import __version__
 from qibo.backends import einsum_utils
+from qibo.backends.npmatrices import NumpyMatrices
 from qibo.backends.numpy import NumpyBackend
 from qibo.config import raise_error
 
 
+class JaxMatrices(NumpyMatrices):
+    def __init__(self, dtype):
+        super().__init__(dtype)
+        import jax  # pylint: disable=import-error
+        import jax.numpy as jnp  # pylint: disable=import-error
+
+        self.jax = jax
+        self.np = jnp
+
+    def _cast(self, x, dtype):
+        return self.np.array(x, dtype=dtype)
+
+    def Unitary(self, u):
+        return self._cast(u, dtype=self.dtype)
+
+
 class JaxBackend(NumpyBackend):
```

Comment:
I would probably rewrite the `execute_circuit` and `apply_gate` functions of the backend to properly make use of the jit decorator